pyproject.toml

```diff
@@ -38,6 +38,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
Also remove the 3.8 classifier above and bump `requires-python`.

Reply:
Good catch!